### PR TITLE
fix lint and build issues

### PR DESCRIPTION
**Pull request by tzdybal**
## Overview

This PR fixes build issues. local-celestia-devnet is used for unit tests.

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: client_test.go [hunk 1]
```diff
@@ -1,26 +1,108 @@
...
+	resource, err := pool.Run("ghcr.io/rollkit/local-celestia-devnet", "c0b729d", []string{})
+	if err != nil {
+		t.Failf("Could not start resource", "error: %v\n", err)
+	}
+	t.resource = resource
+
+	// exponential backoff-retry, because the application in the container might not be ready to accept connections yet
+	pool.MaxWait = 60 * time.Second
+	if err := pool.Retry(func() error {
+		resp, err := http.Get("http://localhost:" + resource.GetPort("26659/tcp") + "/balance")
```
Comment (tuxcanfly) [inline client_test.go:51]:
```suggestion
		resp, err := http.Get(fmt.Sprintf("http://localhost:%s/balance", resource.GetPort("26659/tcp")))
```

Comment (tuxcanfly) [reply client_test.go:51]: I feel this is easier to read.

**Review by tuxcanfly — DISMISSED** [review]
OK.

File: client_test.go [hunk 2]
```diff
@@ -1,26 +1,108 @@
...
+	}
+}
+
+func TestIntegrationTestSuite(t *testing.T) {
+	suite.Run(t, new(TestSuite))
+}
+
+func (t *TestSuite) TestClient() {
+	fmt.Println("port:", t.resource.GetPort("26658/tcp"))
+	client, err := NewClient(context.Background(), "http://localhost:"+t.resource.GetPort("26658/tcp"), t.token)
```
Comment (tuxcanfly) [inline client_test.go:93]: Same here.

**Review by tuxcanfly — APPROVED** [review]
OK.

**Review by S1nus — APPROVED** [review]

**Pull request merged**